tests/usdt_sample/scripts/latency.c

```diff
@@ -0,0 +1,127 @@
+#include <linux/blkdev.h>
+#include <uapi/linux/ptrace.h>
+
+// The following methods were taken from the verbose output of argdist.py.
+// Function names were adjusted to match the BPF function name expectations.
+static inline int _bpf_readarg_trace_operation_start_1(struct pt_regs* ctx, void* dest, size_t len)
```

Comment:
I agree that we probably should add comments to describe the transformation.

The best non-code doc is probably from the following long commit message:

```
commit 3e39a08a4fd5eec7fac6b8910b190087e76805b4
Author: Sasha Goldshtein <goldshtn@gmail.com>
Date:   Thu Mar 24 08:39:47 2016 -0700

    argdist, trace, and tplist support for USDT probes

    ......
```
Basic idea:
    When you enable a tracepoint probe like:
    usdt_ctx.enable_probe(probe="operation_start", fn_name="trace_operation_start")
    The library will read the binary .note.stapsdt section to find the structure for the probe
    "operation_start", and generate functions to read its structure fields.
    In the above example, the first argument can be read through function:
    _bpf_readarg_trace_operation_start_1, and the second argument can be read from
    _bpf_readarg_trace_operation_start_2, etc.

in the BPF program, if you specify:
    bpf_usdt_readarg(2, ctx, &input)
It means you try to read the second field of the attached probe. The call be translated into
_bpf_readarg_trace_operation_start_2.